src/sync/sync.types.ts

```diff
@@ -14,9 +14,9 @@ export interface TaskProcessors {
 		gitHubInstallationClient: GitHubInstallationClient,
 		jiraHost: string,
 		repository: Repository,
-		cursor?: string | number,
-		perPage?: number,
-		messagePayload?: BackfillMessagePayload
+		cursor: string | number | undefined,
```

Comment:
What's the reason for this change? I don't think cursor is a required arg. There are multiple places in the code where we are setting the values for cursor/perPage if it isn't passed e.g. in `getBuildTask` for cursor -> `cursor: string | number = 1`

Reply:
Just typing @rachellerathbone , I'm removing optionality: we always pass over perPage and messagePayload

Reply:
also, typescript would've yelled if something was wrong, however tests are green ...

Reply:
lol you put a lot of faith in our tests.